<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Bioinformatics_Team</title>
<link rel="stylesheet" href="http://necolas.github.io/normalize.css/3.0.2/normalize.css" />
<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML"></script>
</head>
<body><div class="container"><h1 id="bioinformatics-core-team-mrc-csc">Bioinformatics core team (MRC CSC)</h1>

<p>The Bioinformatics team provides Bioinformatics analysis, support, training and software to help maintain high quality computational biology research within the Clinical Sciences Centre.</p>

<p>The core Bioinformatics team consists of 3 section bioinformaticians working within their relevant sections and alongside their academic advisors.</p>

<ul>
<li>Head Of Bioinformatics - Thomas Carroll <a href="thomas.carroll@csc.mrc.ac.uk">thomas.carroll@csc.mrc.ac.uk</a></li>
<li>Epigenetics - Gopuraja Dharmalingham <a href="thomas.carroll@csc.mrc.ac.uk">gopuraja.dharmalingam@csc.mrc.ac.uk</a></li>
<li>Genes And Metabolism - Sanjay Khadayate <a href="sanjay.khadayate@csc.mrc.ac.uk">sanjay.khadayate@csc.mrc.ac.uk</a></li>
<li>Integrative Biology - TBD </li>
</ul>

<p>The bioinformatics team works closely with both Genomics and Proteomics centres to assist in the delivery of high quality genomics and proteomics data.</p>

<hr>



<h2 id="experimental-design">Experimental Design</h2>

<blockquote>
  <p>To call in the statistician after the experiment is done may be no more than asking him to perform a post-mortem examination: he may be able to say what the experiment died of. <br>
  <em>Ronald Fisher</em></p>
</blockquote>

<p>The bioinformatics team provides advice on experimental design and analysis planning at all stages of an experiment.  <br>
Working with the experimentalist(s) alongside Proteomics and/or Genomics cores at early stages in the design allow for the bioinformatics core to assist in the construction of appropriate experimental designs and to ensure the appropriate power may be achieved. </p>

<p>If you would like to consult on the design or required power of upcoming or present experiments please contact Thomas Carroll <a href="thomas.carroll@csc.mrc.ac.uk">thomas.carroll@csc.mrc.ac.uk</a>.</p>



<h2 id="analysis-support">Analysis Support</h2>

<p>The bioinformatics team provides analysis support for all biological data. The team has a large pool of experience in analysis support from initial data processing and QC through to publication. </p>

<p>As a team, we have analysed more than 1000 high throughput sequencing datasets (including ChIP-seq, ChIP-exo,  RNA-seq, MNase-seq, shRNA-seq and whole genome resequencing) and have contributed to many peer-reviews publications from the CSC.</p>

<p>If you have a question about present or future analysis or require analysis support please contact the relevant section bioinformatician or Thomas Carroll <a href="thomas.carroll@csc.mrc.ac.uk">thomas.carroll@csc.mrc.ac.uk</a>.</p>

<blockquote>
  <p><strong>Reproducibility of Research</strong> <br>
  The bioinformatics team works to ensure that all data and analyses are not only of high quality but also <strong>reproducible</strong> by external examiners. <br>
  To ensure this, the bioinformatics team implements tracking of analysis project using Redmine and controls versions and analysis methods using methods such as <a href="http://yihui.name/knitr/">knitr</a> and <a href="http://rstudio.github.io/packrat/">packrat</a> R libraries.</p>
</blockquote>



<h2 id="software">Software</h2>

<p>The Bioinformatics team focuses on the development of Bioinformatics software for high-through analysis, quality control and visualization of biological data relevant to our work at the CSC. All sofware developed is publically available through our github site or from online repositories</p>

<p>We currently maintain 2 analysis pipeline (ChIP-seq and RNA-seq) available on our Github site, to allow for the rapid and reproducible processing and initial analysis of these common data types.</p>

<p>We also developed and maintain two Bioconductor packages within the team:</p>

<ul>
<li><strong>ChIPQC</strong> - This package deals with the quality control of ChIP-seq/ChIP-exo data using common and novel metrics of ChIP-seq quality. <a href="http://bioconductor.org/packages/release/bioc/html/ChIPQC.html">ChIPQC Bioconductor Site</a></li>
<li><strong>tracktables</strong> - This package provides methods to produce HTML reports and markdown reports containing IGV (Integrative Genome Viewer) linked tables. <a href="http://bioconductor.org/packages/release/bioc/html/tracktables.html">tracktables Bioconductor Site</a></li>
<li><strong>sogi</strong> - This package provides methods to Summarise signal (ChIP-seq/motif occurence etc) Over Genomic Intervals. Currently in development, the package is available from Github on request,</li>
</ul>



<h2 id="training">Training</h2>

<p>The Bioinformatics team provides ad-hoc training for biologists as well as training courses covering topics such as ChIP-seq analysis, data visualisation etc.</p>

<p>All course material is made publicly available and in the new year, the core will be running training for basic ChIP-seq and RNA-seq analysis. For details on upcoming courses see our <a href="https://github.com/ThomasCarroll/bioinformatics-training">Github site</a> or contact Thomas Carroll <a href="thomas.carroll@csc.mrc.ac.uk">thomas.carroll@csc.mrc.ac.uk</a>.</p>

<blockquote>
  <p>Current training material can be found at <br>
  <a href="https://github.com/ThomasCarroll/bioinformatics-training">https://github.com/ThomasCarroll/bioinformatics-training</a></p>
</blockquote>

<p>As training in Bioinformatics methods, the Bioinformatics team can offer advice on the use of local compute resources (i.e. best practice in using R in cluster compute environment).  <br>
If you would like to have advice on using our local computing cluster please contact Thomas Carroll <a href="thomas.carroll@csc.mrc.ac.uk">thomas.carroll@csc.mrc.ac.uk</a>.</p>

<blockquote>
  <p><strong>Note:</strong>  A later version of this document may exist on our <a href="http://mrccsc.github.io/">webpage</a></p>
</blockquote></div></body>
</html>
